### ISSUE TITLE
 can use   setFragmentResultListener  & setFragmentResult ?

### ISSUE THREAD

**Issue by CMingTseng** [xiazunyang/Mortar#1]
Dear Sir 

can ref this https://developer.android.com/guide/fragments/communicate?hl=zh-tw

https://developer.android.com/static/images/guide/fragments/fragment-a-to-b.png?hl=zh-tw

THX

**Comment by xiazunyang** [comment]
No, There was no setFragmentResult API when this project appeared.
